code/modules/mob/living/simple_mob/simple_mob.dm

```diff
@@ -222,6 +222,7 @@
 /mob/living/simple_mob/Login()
 	. = ..()
 	to_chat(src,"<b>You are \the [src].</b> [player_msg]")
+	verbs |= /mob/living/simple_mob/proc/hunting_vision //So that maint preds can see prey through walls, to make it easier to find them.
```

Comment:
As I've later written, I believe making this verb addition happen on /mob/living/simple_mob/vore might be a better idea as we're intending for this to be a scene tool. This likely will not cause us issues, but I believe it's best to restrict it to the scene mob subtypes by default.

This is more or less for sake of downstreams.

Alternatively, you could add a var/ShouldHaveThermals = FALSE on simple_mob then change this to

`
if(shouldHaveThermals)
   verbs |= /mob/living/simple_mob/proc/hunting_vision
`

Then modify /mob/living/simple_mob/vore to have this var as true.

I think this might be the best approach to this, as we can fine tune down the line whether we want a mob to have it or not. Downstream can also make their own edits.